beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/path/aws/S3PathCleaner.java

```diff
@@ -77,8 +77,10 @@ public void cleanupPath(HousekeepingPath housekeepingPath) {
         }
       } catch (Exception e) {
         log.warn("Sentinel file(s) could not be deleted", e);
+        return false;
```

Comment:
I guess this case was left so that we don't fail the path due to a sentinel file error (which are zero bytes anyway). But perhaps it is right to. What do others think about this?